services/server/test/integration/verification-handlers/verify.stateless.spec.ts

```diff
@@ -1052,4 +1052,66 @@ describe("/", function () {
       "perfect",
     );
   });
+
+  it("should verify a vyper contract resulting in a partial match", async () => {
```

Comment:
But all the tests regarding APIs for stateless verification are there... At this point we can split also all the other tests into their own file for consistency

Reply:
What I meant is that `verify.stateless.spec.ts` would test anything on the `/verify` endpoint, and `vyper.stateless.spec.ts` would test anything on the `/very/vyper` endpoint. 

I think this makes sense since we have `verify.stateless.handlers.ts` and `vyper.stateless.handlers.ts`.

How would you split it?

Reply:
we'll split the test file after merging